README.md

```diff
@@ -1,264 +1,505 @@
-- pattern-lab/
-  - config/config.yml (Pattern Lab configuration)
-  - public/ (Where Pattern Lab compiles too, it's just static HTML)
-  - composer.json (run `composer update` next to this to update dependencies)
-- scss/ - Sass files that aren't really tied to a component, so not in the above location.
-- js/ - all js files here and transpiled by Babel and combined into a single `dest/script.js` file.
-- images/icons/src/ - all SVGs here are combined into font icons and have classes and Sass mixins made for each based on file name. See `atoms/images/icons` in Pattern Lab.
-- dest/ - Many compiled assets end up in here like CSS, JS, Font Icons, and any doc systems like [SassDoc](http://sassdoc.com).
-- templates/ - Drupal twig templates. These often will `include`, `embed`, or `extend` the Twig templates found in Pattern Lab like this: `{% include "@molecules/branding/branding.twig" with { url: path('<front>') } %}`. We keep the components in Pattern Lab "pure" and ignorant of Drupal's data model and use these templates to map the data between the two. Think of these as the Presenter templates in the [Model View Presenter](https://en.wikipedia.org/wiki/Model–view–presenter) approach. Also, Drupal Twig templates that have nothing to do with Pattern Lab go here.
-- gulpconfig.yml - Configuration for all the gulp tasks, a ton can be controlled here.
+There is a very clear role for each in the component system of NAMETBD. In the `button` component featured above in [Anatomy of a Component](#anatomy-of-a-component), note this import:
 
-### IDE/Text Editor Setup
+```javascript
+// source/_patterns/01-atoms/button/_index.js
+...
+import './_button.scss';
+...
 
-- Install an EditorConfig plugin
-- Ignore the indexing of these directories:
-  - `node_modules/`
-  - `bower_components/`
-  - `dest/`
-  - `pattern-lab/public/`
-  - `pattern-lab/vendor/`
+```
 
----
+Looking into `source/_patterns/01-atoms/button/_button.scss` reveals:
 
-# Details
+```scss
+@import '../../00-base/config'; // DOES NOT OUTPUT CSS!
 
-## Pattern Lab
+$btn-border-radius: 0.25rem;
+@import "~bootstrap/scss/buttons"; // OUTPUTS CSS!
 
-Refer to the [Pattern Lab Documentation](http://patternlab.io/docs) for extensive info on how to use it. This theme starter is a custom Pattern Lab 2 *Edition* that is heavily influenced by the [Drupal Edition of Pattern Lab](https://github.com/pattern-lab/edition-php-drupal-standard) and uses the Twig engine to bring it inline with Drupal 8's use of Twig.
+.custom-class {
+  color: red;    // OUTPUTS CSS!
+}
+```
 
-### Folder Structure Differences
+This approach to component styes allows sharing non-printing Sass **configuration**, while also ensuring our component prints its custom CSS exactly once. We can now safely `@import 'atoms/button;` anywhere in our other javascript components as many times as needed and there will be no duplicate CSS output for buttons!
```

Comment:
Was this the solution to the CSS duplication? I seem to remember a PostCSS discussion, but that was a week or so ago and with things moving fast I could have missed an update/breakthrough on this.

Reply:
Yeah, this is the architectural approach that doesn't need an after-the-fact de-dupe process. It's definitely not as fire-and-forget, but pretty stable.